datalad/metadata/search.py

```diff
@@ -743,10 +758,20 @@ def __init__(self):
             ))
 
     def get_query(self, query):
-        # cmdline args might come in as a list
-        if isinstance(query, list):
-            query = u' '.join(query)
-        return query.lower()
+        query = assure_list(query)
+        simple_fieldspec = re.compile(r"(?P<field>\S+?):(?P<query>.*)")
+        quoted_fieldspec = re.compile(r"'(?P<field>[^']+?)':(?P<query>.*)")
```

Comment:
I guess this should be adjusted to allow for saying explicitly that `:` is a part of "direct" (just `q`), non-fieldspec query.  I guess we could allow for it to be escaped `\`, so above ones could then become instead of `\S+?` a `\S*?[^\]` , and we would then need to "unscape" `\:` ?  but then what if we are looking for `\:`? (I guess would need a lookbehind that previous one is not `\` and then we convert `\\` to `\`
?

Reply:
I though about it, and I find a leading colon to be more natural to indicate ALL FIELDS. I.e. `:term: with colon` will search for `term: with colon`. and `:this` is identical to `this`. If one needs to search for something starting with a colon, this will do: `::term`

Reply:
Awesome